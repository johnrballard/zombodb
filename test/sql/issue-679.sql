alter index idxso_posts set (options='user_data:(owner_user_id=<public.so_users.idxso_users>id)');
alter index idxso_posts set (max_terms_count = 10131072);
select zdb.count('so_posts', '(not user_data.id:-1 or owner_user_id = null)');
select zdb.count('so_posts', '(not user_data.id:-1 or owner_user_id = null) or user_data.id:-1');
select zdb.count('so_posts', '((not user_data.id:-1 or owner_user_id = null) or user_data.id:-1 and user_data.id:-1) and (not user_data.id:-1 or owner_user_id = null)');
alter index idxso_posts reset(options);
alter index idxso_posts reset(max_terms_count);
