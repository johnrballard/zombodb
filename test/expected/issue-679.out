alter index idxso_posts set (options='user_data:(owner_user_id=<public.so_users.idxso_users>id)');
alter index idxso_posts set (max_terms_count = 10131072);
select zdb.count('so_posts', '(not user_data.id:-1 or owner_user_id = null)');
 count  
--------
 164867
(1 row)

select zdb.count('so_posts', '(not user_data.id:-1 or owner_user_id = null) or user_data.id:-1');
 count  
--------
 165240
(1 row)

select zdb.count('so_posts', '((not user_data.id:-1 or owner_user_id = null) or user_data.id:-1 and user_data.id:-1) and (not user_data.id:-1 or owner_user_id = null)');
 count  
--------
 164867
(1 row)

alter index idxso_posts reset(options);
alter index idxso_posts reset(max_terms_count);
